### DISCUSSION TITLE
A possibility to adjust the key LED brightness

### DISCUSSION THREAD

**Discussion by lobodpav** [solokeys/solo2#104] [Ideas]
The Key glows like crazy at night.

It would be wonderful to have a chance to adjust the brightness or turn it off completely, using the CLI.

**Comment by nickray** [comment]
Yeah I support this :) On my dev firmware for the next release it's a milder red breathing; to make things fully configurable will need a bit of work (although just "everything off" shouldn't be too hard). There's kind of a "visual language" (which doesn't work on the secure keys due to a bug anyway), distinguishing between:
- key at rest
- key busy calculating/processing something
- user presence required
- winking

**Reply by lobodpav** [reply]
Sounds lovely!

**Comment by coelner** [comment]
some minor thoughts about LED brigthness, if the solo2 is used with the rubber sleeve, you are never sure to have the LED 'on top' and to see it.
- place another smb led on the modules back and cut out a hole into the framing pcb, lost of the flat backside
- cut a hole into the module, place another smb led upsidedown, lost of the flat backside , but maybe good enough to fit into the framing pcb
- place a marking on the usb plug to show which side up
- change the rubber sleve to have just one buckle
- -or just ignore it, because the LED is on when plugged in and the device is easily to turnaround :-)
